packages/office-ui-fabric-react/src/components/Shimmer/ShimmerGap/ShimmerGap.styles.ts

```diff
@@ -0,0 +1,39 @@
+import { IShimmerGapStyleProps, IShimmerGapStyles } from './ShimmerGap.types';
+import { IStyleSet, getGlobalClassNames, HighContrastSelector } from '../../../Styling';
+
+const GlobalClassNames = {
+  root: 'ms-ShimmerGap-root'
+};
+
+export function getStyles(props: IShimmerGapStyleProps): IShimmerGapStyles {
+  const { height, widthInPercentage, widthInPixel, borderStyle, theme } = props;
+
+  const { palette } = theme;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const styles: IStyleSet = !!borderStyle ? borderStyle : {};
+  const ACTUAL_WIDTH = widthInPercentage ? widthInPercentage + '%' : widthInPixel ? widthInPixel + 'px' : '10px';
```

Comment:
you don't need to do this!

just pass in `width: number | string;` A number will convert to a pixel unit!

Reply:
fixed